packages/ChangesReporting/Output/WorkerOutputFormatter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\ChangesReporting\Output;
+
+use Nette\Utils\Json;
+use Rector\ChangesReporting\Annotation\RectorsChangelogResolver;
+use Rector\ChangesReporting\Contract\Output\OutputFormatterInterface;
+use Rector\Core\Exception\ShouldNotHappenException;
+use Rector\Core\ValueObject\Configuration;
+use Rector\Core\ValueObject\Error\SystemError;
+use Rector\Core\ValueObject\ProcessResult;
+use Rector\Parallel\ValueObject\Bridge;
+
+final class WorkerOutputFormatter implements OutputFormatterInterface
```

Comment:
I have introduced this new formatter, so I can easily differentiate between a user requesting json format and a user requesting a different format, but rector is internally using json format while delegating to worker processes.

that was necessary for this PR to implement the pre-existing behaviour